tests/functional/pages/surveys/census/household/household-relationships.page.js

```diff
@@ -8,68 +8,68 @@ class HouseholdRelationshipsPage extends MultipleChoiceWithOtherPage {
     super('household-relationships')
   }
 
-  clickHouseholdRelationshipsAnswerHusbandOrWife(instance = 0) {
-    browser.element('[id="household-relationships-answer-' + instance + '-0"]').click().pause(300)
+  clickHouseholdRelationshipsAnswerHusbandOrWife(instance = 0, value) {
+    browser.selectByValue('[id="household-relationships-answer-' + instance + '"]', value)
     return this
   }
 
-  clickHouseholdRelationshipsAnswerSameSexCivilPartner(instance = 0) {
-    browser.element('[id="household-relationships-answer-' + instance + '-1"]').click().pause(300)
+  clickHouseholdRelationshipsAnswerSameSexCivilPartner(instance = 0, value) {
```

Comment:
It seems wrong the have lots of `clickHouseholdRelationshipsAnswerXXXXX` and the pass the value in to select. I means all of the other `clickHouseholdRelationshipsAnswerXXXXX` methods are redundant. We should do one of the other.

1) Have one method where the value is passed in or
2) Have one method per answer type.

To be consistent I'd suggest the latter.